plugins/wazuh_check_updates/server/services/cronjob/jobSchedulerRun.ts

```diff
@@ -0,0 +1,16 @@
+import cron from 'node-cron';
+import { DEFAULT_SCHEDULE } from '../../../common';
+import { getSettings } from '../settings';
+import { getUpdates } from '../updates';
+
+export const jobSchedulerRun = async () => {
+  try {
+    const settings = await getSettings();
+
+    cron.schedule(settings?.schedule || DEFAULT_SCHEDULE, () => getUpdates());
+  } catch (error) {
+    const message = error instanceof Error ? error.message : error;
+    console.log('wazuh-check-updates:jobSchedulerRun', message);
```

Comment:
**nitpick**: use the logging of the platform.

Reply:
Done